test/watch.js

```diff
@@ -126,10 +131,11 @@ describe('gulp', function() {
         }, timeout);
 
         // it works if it calls the task
-        var watcher = gulp.watch(tempFile, {debounceDelay:timeout/2}, [task1,task2]);
+        var watcher = gulp.watch(tempFile, {debounceDelay: timeout / 2},
+          [task1, task2]);
```

Comment:
Imo it would be better to just move the object into a variable:

```
var config = {debounceDelay: timeout / 2};
var watcher = gulp.watch(tempFile, config, [task1, task2]);
```